app/helpers/support/allocation_batch_jobs_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Support::AllocationBatchJobsHelper
+  def aggregate_allocation_change(jobs)
+    jobs.sum { |job| job.applied_allocation_delta.to_i }
+  end
+
+  def processed_jobs(jobs)
+    "#{jobs.count(&:processed)} / #{jobs.size}"
+  end
```

Comment:
As discussed in Slack, it would be interesting to see how these helper methods perform for batches of 20,000 or even 40,000 jobs.

Reply:
I guess they all would be increasing on every refresh of the page as sidekiq receives and executes more and more jobs.